patches/chrome-utility-importer-bookmark_html_reader.cc.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/chrome/utility/importer/bookmark_html_reader.cc b/chrome/utility/importer/bookmark_html_reader.cc
+index 81985a841756925087beb179c9f8314caca2a27b..6bf102cb1aeee821d248c6c15efde9b2143b4a32 100644
+--- a/chrome/utility/importer/bookmark_html_reader.cc
++++ b/chrome/utility/importer/bookmark_html_reader.cc
+@@ -17,7 +17,9 @@
+ #include "base/strings/utf_string_conversions.h"
+ #include "base/time/time.h"
+ #include "chrome/common/importer/imported_bookmark_entry.h"
++#if !BUILDFLAG(IS_ANDROID)
```

Comment:
just fyi, instead of such way, you can use chromium_src override  for `src/brave/chromium_src/chrome/utility/importer/bookmark_html_reader.cc` and put there header guard
```c++
#if BUILDFLAG(IS_ANDROID)
#define CHROME_UTILITY_IMPORTER_FAVICON_REENCODE_H_
#endif
```
Not applicable exactly here since https://github.com/brave/brave-core/pull/17821/files#r1160610690